pallets/runtime/src/bridge.rs

```diff
@@ -167,6 +185,9 @@ decl_event! {
         FrozenTx(BridgeTx<AccountId, Balance>),
         /// Notification of unfreezing a transaction.
         UnfrozenTx(BridgeTx<AccountId, Balance>),
+        /// A vector of timelocked balances of a recipient, each with the number of the block in
+        /// which the balance gets unlocked.
+        TimelockedBalancesOfRecipient(Vec<(BlockNumber, Balance)>),
```

Comment:
I'll check if using reserved balance is a better option. My feeling was that it was an added complexity because we still need to maintain the state for each particular transaction. Hence adding reserved balance wouldn't add anything apart from the requirement to keep it in sync with the transaction state.

Reply:
To summarise the discussion, `TimelockedBalancesOfRecipient` is just an event and not the place where we store the balances. Storing all reserved balances of a recipient in the balances module would not be informative (the unlock block numbers would be missing) and is only possible as an additional feature. If desired, the feature can be implemented as a separate task. It would require comprehensive testing to ensure we are not introducing inconsistencies while duplicating transition state into the balances module. I think this feature is not necessary and rather depends to the product req's.

Reply:
As long as we don't need to included time-locked poly in rewards calculation, the current approach is fine imo.

Reply:
I don't think they should be included in the rewards calculations, so current approach looks good.